content/blog/2020-07-09-cloud-update/index.Rmarkdown

```diff
@@ -0,0 +1,128 @@
+
+-   Classes can be organized in workspaces
+-   Members can be assigned different roles: instructor, TA, student
+-   Projects can be public or private
+-   Projects can be made into assignments
+-   Students can make copies of projects created by the instructor, with the necessary environment automatically replicated.
+-   Instructor can peek into and edit student projects
+
+In addition to teaching data science, RStudio Cloud is a great platform for doing data science as well. The ability to share projects makes it easy for researchers from different groups or institutions to collaborate, and is useful for any data science team that wants to build and share data science work, without having to maintain their own IT infrastructure.
+
+To learn more:
+
+-   Visit <https://rstudio.com/products/cloud>,
+-   Check out our recent webinar on [Teaching R Online with RStudio Cloud](https://rstudio.com/resources/webinars/teaching-r-online-with-rstudio-cloud/),
+-   Sign up at <https://rstudio.cloud/> and get started for free.
+
+## What's changing with RStudio Cloud?
+
+After an extensive alpha/beta period, RStudio Cloud will be generally available for use and purchase in the fall. (Until then, you can still sign up for the Cloud Free plan - the usage limits are currently not enforced). While RStudio Cloud will always offer a free plan for individuals, beginning August 3, 2020, we will also offer paid premium plans for individuals, instructors, students and organizations. Learn more on our [Plans & Pricing page](https://rstudio.cloud/plans/).
+
+## Why are we making this change?
+
+At RStudio, we offer our professional products ([RStudio Server Pro](https://rstudio.com/products/rstudio-server-pro/), [RStudio Connect](https://rstudio.com/products/connect/), [RStudio Package Manager](https://rstudio.com/products/package-manager/)) for free for instructional use. You can read more about these options [here](/teach/tools/#computing-infrastructure).
+
+However, with all of those pro products, it is then up to the user to shoulder any costs associated with hosting and maintaining them. This often requires quite a lot of leg work, and usually there is an organizational investment in the infrastructure needed to use the pro products. RStudio Cloud is different. We want to make Cloud affordable for instructional use, but we need to cover our ongoing hosting costs so that we can continue to offer the service. We've priced Cloud for instructional use as low as we can and still cover our hosting costs---and we're not looking at it as a profit generator.
+
+In short: the fees help us offset the cost of operating the service, and will enable us to continue to provide a free plan for individuals to do data science.
+
+## What are my options for teaching classes and workshops?
+
+The pricing plans are available at <https://rstudio.cloud/plans/>:
+
+![](cloud-screenshot.png)
```

Comment:
I don't think screenshot helps here; you can't use it, it's inaccessible, and it duplicates info available at the link.